src/components/Proposal/ProposalDetailsPage.tsx

```diff
@@ -336,7 +336,7 @@ class ProposalDetailsPage extends React.Component<IProps, IState> {
         {this.state.showShareModal ?
           <SocialShareModal
             closeHandler={this.closeShareModal}
-            url={`https://alchemy.daostack.io/dao/${daoState.address}/proposal/${proposal.id}`}
+            url={`https://${targetedNetwork() !== "main" ? `${targetedNetwork()}.` : ""}v1.alchemy.do/dao/${daoState.address}/proposal/${proposal.id}`}
```

Comment:
How about using env var BASE_URL here ?

Reply:
We have a `BASE_URL` in `webpack.prod.config.js` but since we're adding text in the middle it's not usable.

Reply:
not sure why. ?
you can have a  util function `getBaseURL()`

Reply:
Let me try something 👍